creators/models.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
See comments above - let's preserve the 1 model = 1 file structure from above, and shift this into a /models directory in the creators app